buildkitd/buildkitd.go

```diff
@@ -176,11 +174,12 @@ func Start(ctx context.Context, image string, settings Settings, reset bool) err
 		return errors.Wrap(err, "settings hash")
 	}
 	env := os.Environ()
-	cacheMount := fmt.Sprintf("%s:%s:delegated", TempDir, TempDir)
+	cacheMount := fmt.Sprintf("%s:%s:delegated", settings.TempDir, settings.TempDir)
```

Comment:
We could always mount to the same place within the container (eg keep it as `/tmp/earthly` within buildkitd). I would only configure the source. That way we don't have to change the buildkit settings. What do you think?

Reply:
I was thinking about doing it that way too. lets make that change, then it's less variables to pass into the container.

Reply:
fixed.